### DISCUSSION TITLE
Touchable Buttons Not Firing onPress Inside PanGestureHandler

### DISCUSSION THREAD

**Discussion by fredrickreuben** [software-mansion/react-native-gesture-handler#3172] [General]
How are you guys getting Touchables to work inside a GestureDetector, especially with PanGesture? In my case, the buttons inside aren't triggering the onPress function. It's as if the tap isn't being detected.

**Comment by henriquebayer1** [comment]
did you find what was?

**Comment by bunkscene** [comment]
This was pretty annoying for me.  I'm not sure if you have the same setup as me, but I used the two properties below to help alleviate my issues. Using these make the pan not get registered until a certain amount of pixels have been panned, so if you tap, it won't get triggered.  You can adjust the values per your needs.

```tsx
  const panGesture = Gesture.Pan()
    .activeOffsetX([-10, 10])
    .activeOffsetY([-10, 10])
